geometry_msgs/Point gps_bar
geometry_msgs/Point beacons
